### PR TITLE
update profiling page

### PR DESCRIPTION
**Pull request by lizokm**
Update existing Profiling page and add mobile profiling page.

### CODE REVIEW

File: src/docs/product/profiling/index.mdx [hunk 1]
```diff
@@ -22,24 +22,34 @@ Profiling is currently in beta. Beta features are still in-progress and may have
 
 </Note>
 
-Profiling allows you to collect and analyze performance profiles from real user devices in production to give you a complete picture of how your application performs in a variety of environments. It's the next-generation performance monitoring feature for your apps, removing the limitations of manually created tests, by looking at real user experiences. With profiling, you're building in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one.
+Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring] (/product/performance/) capabilities to provide precise code-level visibility into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.
```
Comment (lizokm) [inline src/docs/product/profiling/index.mdx:25]:
```suggestion
Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring](/product/performance/) capabilities to provide precise code-level visibility into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.
```

File: src/docs/product/profiling/index.mdx [hunk 2]
```diff
@@ -22,24 +22,34 @@ Profiling is currently in beta. Beta features are still in-progress and may have
...
 
-![Profiling dashboard displaying a flamegraph](profiling-dashboard.png)
+Sentry profiling supports common platforms for both Mobile and Backend applications:
 
-The typical approach to measuring app performance involves setting up automated performance tests for various user flows that run during continuous integration (CI), or manually profiling hot paths in your app using tools like Apple Instruments or Android Profiler. These are excellent tools, but they have numerous drawbacks:
+- [Mobile app profiling](/product/profiling/getting-started/mobile-app-profiling) makes it possible to collect **profiles from real devices in production**, providing a complete picture of how the app performs in a variety of environments.
```
Comment (lizokm) [inline src/docs/product/profiling/index.mdx:29]:
```suggestion
- [Mobile app profiling] NEED LINK makes it possible to collect **profiles from real devices in production**, providing a complete picture of how the app performs in a variety of environments.
```

File: src/docs/product/profiling/index.mdx [hunk 3]
```diff
@@ -22,24 +22,34 @@ Profiling is currently in beta. Beta features are still in-progress and may have
...
+
+You can go to the Profiling page to see a list of transactions in descending order of execution time for your selected project(s) and also view the three “Slowest Transactions” broken down further with a list of the slowest identified functions. The “Profiles by Percentiles” timeline chart can be used to filter the transaction data by highlighting the desired time period.  Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.
+
+[Image 3]
+
+To zoom in further, select from the list of “Slowest App Functions” and see the associated flame chart for the profile.  (*How to read a flame chart link*)
+
+Selecting one of the transactions will take you to the “Transaction Profile” summary page below.  On this page, you can examine the consolidated flame graph (*How to read flame graph link*) that presents aggregated sample data for the selected transaction.
+
+[Image 4]
```
Comment (lizokm) [inline src/docs/product/profiling/index.mdx:52]:
```suggestion

![profiling-information-span](https://user-images.githubusercontent.com/26800704/234110886-7b3912d1-883f-4227-b40e-62c84965c460.png)

From this view, you can click “View Profile” to zoom in on the flame chart. (How to read a flame chart link)

![profiling-summary-view](https://user-images.githubusercontent.com/26800704/234110914-f99519ba-e469-43e0-9579-6aeee7c3186b.png)


## Profiling Page

You can go to the Profiling page to see a list of transactions in descending order of execution time for your selected project(s) and also view the three “Slowest Transactions” broken down further with a list of the slowest identified functions. The “Profiles by Percentiles” timeline chart can be used to filter the transaction data by highlighting the desired time period.  Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.

![profiling-page-view](https://user-images.githubusercontent.com/26800704/234110952-aca7822e-d55c-4f69-8fc2-8a8989c800ac.png)


To zoom in further, select from the list of “Slowest App Functions” and see the associated flame chart for the profile.  (*How to read a flame chart link*)

Selecting one of the transactions will take you to the “Transaction Profile” summary page below.  On this page, you can examine the consolidated flame graph (*How to read flame graph link*) that presents aggregated sample data for the selected transaction.

![transaction-profile-view](https://user-images.githubusercontent.com/26800704/234111012-56e3e3dd-69e1-4925-9b25-4e1477223aad.png)

```

File: src/docs/product/profiling/index.mdx [hunk 4]
```diff
@@ -22,24 +22,38 @@ Profiling is currently in beta. Beta features are still in-progress and may have
 
 </Note>
 
-Profiling allows you to collect and analyze performance profiles from real user devices in production to give you a complete picture of how your application performs in a variety of environments. It's the next-generation performance monitoring feature for your apps, removing the limitations of manually created tests, by looking at real user experiences. With profiling, you're building in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one.
+Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring](/product/performance/) capabilities to provide precise code-level visibility into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.
```
Comment (lizokm) [inline src/docs/product/profiling/index.mdx:25]:
```suggestion
Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring](/product/performance/) capabilities to provide precise, code-level insight into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.
```

File: src/docs/product/profiling/index.mdx [hunk 5]
```diff
@@ -22,24 +22,38 @@ Profiling is currently in beta. Beta features are still in-progress and may have
...
 
-![Profiling dashboard displaying a flamegraph](profiling-dashboard.png)
+Sentry profiling supports common platforms for both Mobile and Backend applications:
 
-The typical approach to measuring app performance involves setting up automated performance tests for various user flows that run during continuous integration (CI), or manually profiling hot paths in your app using tools like Apple Instruments or Android Profiler. These are excellent tools, but they have numerous drawbacks:
+- [Mobile app profiling] NEED LINK makes it possible to collect **profiles from real devices in production**, providing a complete picture of how the app performs in a variety of environments.
```
Comment (lizokm) [inline src/docs/product/profiling/index.mdx:29]:
```suggestion
- [Mobile app profiling](/product/profiling/mobile-app-profiling/) makes it possible to collect **profiles from real devices in production**, providing a complete picture of how the app performs in a variety of environments.
```

File: src/docs/product/profiling/index.mdx [hunk 6]
```diff
@@ -22,24 +22,38 @@ Profiling is currently in beta. Beta features are still in-progress and may have
...
+## Zooming In From Performance To Profiling
 
-Profiling is designed to address all of these issues:
+Sentry profiling data is structured as an extension of performance transactions.  From the Sentry **Performance** page, you can review high level transaction data from your application to identify suspect spans - operations that may be impacting performance, including slow DB queries, HTTP requests, and so on.  You can click on a span to see profiling information, including the most frequently occurring code path (call stack with with exact line numbers), along with the approximate percentage of time required for that code path.
+
+
+![profiling-information-span](https://user-images.githubusercontent.com/26800704/234110886-7b3912d1-883f-4227-b40e-62c84965c460.png)
+
+From this view, you can click “View Profile” to zoom in on the flame chart. (How to read a flame chart link)
```
Comment (lizokm) [inline src/docs/product/profiling/index.mdx:39]:
```suggestion
From this view, you can click “View Profile” to zoom in on the flame chart. 
```

File: src/docs/product/profiling/index.mdx [hunk 7]
```diff
@@ -22,24 +22,38 @@ Profiling is currently in beta. Beta features are still in-progress and may have
...
+
+![profiling-summary-view](https://user-images.githubusercontent.com/26800704/234110914-f99519ba-e469-43e0-9579-6aeee7c3186b.png)
+
+
+## Profiling Page
+
+You can go to the Profiling page to see a list of transactions in descending order of execution time for your selected project(s) and also view the three “Slowest Transactions” broken down further with a list of the slowest identified functions. The “Profiles by Percentiles” timeline chart can be used to filter the transaction data by highlighting the desired time period.  Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.
+
```
Comment (lizokm) [inline src/docs/product/profiling/index.mdx:47]:
```suggestion
You can go to the **Profiling** page to see a list of transactions in descending order of execution time for your selected project(s) and also view the three “Slowest Transactions” broken down further with a list of the slowest identified functions. The “Profiles by Percentiles” timeline chart can be used to filter the transaction data by highlighting the desired time period.  Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.

```

File: src/docs/product/profiling/index.mdx [hunk 8]
```diff
@@ -22,24 +22,38 @@ Profiling is currently in beta. Beta features are still in-progress and may have
...
+![profiling-page-view](https://user-images.githubusercontent.com/26800704/234110952-aca7822e-d55c-4f69-8fc2-8a8989c800ac.png)
+
+
+To zoom in further, select from the list of “Slowest App Functions” and see the associated flame chart for the profile.  (*How to read a flame chart link*)
```
Comment (lizokm) [inline src/docs/product/profiling/index.mdx:51]:
```suggestion
To zoom in further, select from the list of “Slowest App Functions” and see the associated flame chart for the profile.  
```

File: src/docs/product/profiling/index.mdx [hunk 9]
```diff
@@ -22,24 +22,38 @@ Profiling is currently in beta. Beta features are still in-progress and may have
...
+
+Selecting one of the transactions will take you to the “Transaction Profile” summary page below.  On this page, you can examine the consolidated flame graph (*How to read flame graph link*) that presents aggregated sample data for the selected transaction.
```
Comment (lizokm) [inline src/docs/product/profiling/index.mdx:53]:
```suggestion
Selecting one of the transactions will take you to the “Transaction Profile” summary page below.  On this page, you can examine the consolidated flame graph that presents aggregated sample data for the selected transaction.
```

File: src/docs/product/profiling/mobile-app-profiling.mdx [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+- **We cover the widest possible set of user flows.** Since the data is collected from actual users using your app as they normally would, we can capture all kinds of usage patterns that you may not have thought about when building your own performance tests.
+- **Metrics are calculated using a large number of samples.** We can collect data from up to 100% of your user population in production, allowing us to compute more accurate metrics.
+- **The process is simple.** Integrate the Sentry SDK, enable profiling, ship your app, and let us worry about analyzing performance and detecting regressions between versions. We encourage _testing in production_, which improves developer velocity and reduces the overhead of having to maintain your own tests and tooling. Our dashboard is designed to be approachable to all engineers, and not just people who have a performance engineering background.
+- **The overhead is minimal.** Depending on the platform, the overhead can be lower than 1% of your app’s CPU time.
+
+## Learn More
+
+<PageGrid />
```
Comment (lizokm) [inline src/docs/product/profiling/mobile-app-profiling.mdx:24]:
```suggestion

```

File: src/docs/product/profiling/index.mdx [hunk 11]
```diff
@@ -22,24 +22,38 @@ Profiling is currently in beta. Beta features are still in-progress and may have
 
 </Note>
 
-Profiling allows you to collect and analyze performance profiles from real user devices in production to give you a complete picture of how your application performs in a variety of environments. It's the next-generation performance monitoring feature for your apps, removing the limitations of manually created tests, by looking at real user experiences. With profiling, you're building in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one.
+Profiling offers a deeper level of code visibility on top of traditional tracing, removing the need for manual test instrumentation and enabling you to build in [performance as a feature](https://blog.codinghorror.com/performance-is-a-feature/) from day one. Sentry’s profiling feature builds upon our established [Performance Monitoring](/product/performance/) capabilities to provide precise, code-level insight into application execution in a production environment. This allows you to quickly identify potential performance bottlenecks and visualize the call stack to find hot paths in your code.
```
Comment (shanamatthews) [inline src/docs/product/profiling/index.mdx:25]: > manual test instrumentation

This sounds weird to me. What does manual test instrumentation mean here? @JonasBa can you confirm this is the best way to talk about this?

File: src/docs/product/profiling/index.mdx [hunk 12]
```diff
@@ -22,24 +22,38 @@ Profiling is currently in beta. Beta features are still in-progress and may have
...
+From this view, you can click “View Profile” to zoom in on the flame chart. 
+
+![profiling-summary-view](https://user-images.githubusercontent.com/26800704/234110914-f99519ba-e469-43e0-9579-6aeee7c3186b.png)
+
+
+## Profiling Page
+
+You can go to the **Profiling** page to see a list of transactions in descending order of execution time for your selected project(s) and also view the three “Slowest Transactions” broken down further with a list of the slowest identified functions. The “Profiles by Percentiles” timeline chart can be used to filter the transaction data by highlighting the desired time period.  Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.
+
+![profiling-page-view](https://user-images.githubusercontent.com/26800704/234110952-aca7822e-d55c-4f69-8fc2-8a8989c800ac.png)
```
Comment (shanamatthews) [inline src/docs/product/profiling/index.mdx:48]:
```suggestion
![profiling-page-view](profiling-page-view.png)
```

**Review by shanamatthews — APPROVED** [review]

**Pull request merged**